Test/Tests/Component/AgentRestart.Tests.ps1

```diff
@@ -124,12 +124,9 @@ Test-WithRetries 3 {
             )]
             $ContrailNetwork = $MultiNode.NM.AddOrReplaceNetwork($null, $Network.Name, $Subnet)
 
-            [Diagnostics.CodeAnalysis.SuppressMessageAttribute(
-                "PSUseDeclaredVarsMoreThanAssignments",
-                "FileLogSources",
-                Justification="It's actually used."
-            )]
-            $FileLogSources = New-ComputeNodeLogSources -Sessions $MultiNode.Sessions
+            [LogSource[]] $LogSources = @()
+            $LogSources += New-EventLogLogSource -Sessions $MultiNode.Sessions -EventLogName "Application" -EventLogSource "Docker"
```

Comment:
I'm not attached to this idea but maybe initialization of docker log should be done in New-ComputeNodeLogSources?